schemas/pulse-job.yml

```diff
@@ -55,13 +80,22 @@ properties:
   display:
     type: "object"
     properties:
-      jobSymbol:
-        title: "jobSymbol"
-        type: "string"
-        # spaces and "?" are not valid for job symbols
-        pattern: "^[A-Za-z0-9._-]+$"
-        minLength: 1
-        maxLength: 25
+      anyOf:
```

Comment:
`anyOf` doesn't work this way...

Wouldn't you want both `jobSymbol` and `chunkId` anyways?
It's just we can facilicate more than one with the same `jobSymbol` as long as `chunkId` is updated.